### DISCUSSION TITLE
Load Test

### DISCUSSION THREAD

**Discussion by firatalkis** [ConSol-Monitoring/snclient#34] [General]
Hi,

Could you had a chance to snclient load testing?

For example, snclient how many can running job per second? or how many nrpe requests per second can snclient accept?

Thanks,
Kind regards.

**Comment by sni** [comment]
We haven't done any load testing. Overhead should be pretty small and depends heavily on which check you do.
You you explain the use case here? What is the point of doing hundreds of checks per seconds on the same agent?

**Reply by firatalkis** [reply]
first of all thank you for your quick replies.

I use the monitoring agent as a jump server in our monitoring architecture and expect it to run 50 jobs at a time.

Normally, it is considered to install a monitoring agent on each server and run 5-10 jobs, but we do not use this method.

I don't know, maybe the method we use is not suitable for monitoring agent:)

we monitor all servers with agentless method but sometimes we need monitoring agent for the some controls.

**Comment by sni** [comment]
well, so far this agent has no remote monitoring capabilities. But there might be some in the future.
How does your agentless monitoring look like?

**Reply by firatalkis** [reply]
Agentless monitoring is provide easy and quickly management of monitoring system.

The all monitoring control scripts of servers are in the only monitoring's server so target server does not have any scripts or agent for monitoring tools.
